ce/mobile-app/mobile-offline-capabilities.md

```diff
@@ -27,7 +27,8 @@ search.app:
 Before you set-up the mobile app in offline mode be sure to read through the following capabilities, tips,  and limitations. 
 
 These tables and corresponding commands are available in offline mode.
-
+  > [!NOTE]
+  > All unsupported commands are hidden in offline.
```

Comment:
Add a link to documentation for the commands available in offline 

https://docs.microsoft.com/dynamics365/mobile-app/mobile-offline-capabilities

Reply:
the note is right before the list of commands that are available in offline.